README.md

```diff
@@ -62,7 +62,9 @@ Start command will start application on 2 ports that need to be exposed to publi
 
 ### Required flags
 
-`--auth-secret` authentication secret used for generating tokens
+`--auth-secret` - authentication secret used for generating tokens
+
+`--private-key` - loadbalancers wallet private key, used for sending founds on payout
```

Comment:
can you open issue to load this from keystore or at least from file?
Because loading like this, private key will be saved in bash history.

Not urgent anyways

Reply:
I opened an issue for this #140